packages/haul-ram-bundle-webpack-plugin/runtime/bootstrap.js

```diff
@@ -1,20 +1,8 @@
-function bootstraper(globalScope, mainId, moduleMappings) { // eslint-disable-line
+function bootstraper(globalScope, bundleName, mainId, moduleMappings) { // eslint-disable-line
   globalScope.__BUNDLE_START_TIME__ = globalScope.nativePerformanceNow
     ? globalScope.nativePerformanceNow()
     : Date.now();
 
-  var ID_MASK_SHIFT = 16;
-  var LOCAL_ID_MASK = ~0 >>> ID_MASK_SHIFT;
-
-  function unpackModuleId(moduleId) {
-    var segmentId = moduleId >>> ID_MASK_SHIFT;
-    var localId = moduleId & LOCAL_ID_MASK;
-    return {
-      segmentId: segmentId,
-      localId: localId,
-    };
-  }
```

Comment:
I'm wondering if we should support both `localId`, `segmentId` and `bundleName` if provided.